saviSimulation/savi/jason_processing/SAVIWorld_model.java

```diff
@@ -66,7 +70,8 @@
 
 	private Button playButton, stopButton;
 	private PShape ugvImage, treeImage, houseImage, threatImage, play, pause, restart, antennaImage, uavImage;
-
+	private PImage bg;
```

Comment:
Is bg short for backGround? If so, I suggest using the name backGround for the variable name as it is easier to read in a hurry.

Reply:
Fixed by @cruizm